server/game/drawcard.js

```diff
@@ -300,6 +300,9 @@ class DrawCard extends BaseCard {
     }
     
     getAttachmentsByKeywords(keywords) {
+        if(!this.attachments) {
```

Comment:
Yeah, I was getting an error here during setup - maybe because I put The Badger in my starting posse for ease of testing? I don't recall the exact phase of the game now, but it was happening reliably on game start until I added this check.